Clean code

yal0 opened the pull request:
### Приветствую, Reviewer 1. Задание было сделано на прошлом курсе, тогда в требованиях к коммитам нужно было писать полное название пунктов. Ссылка на репозиторий с коммитами в новом формате [https://github.com/yal0/clean-code/pull/1](https://github.com/yal0/clean-code/pull/1)

### Спасибо за проверку :)



1. Task: [https://github.com/rolling-scopes-school/tasks/blob/master/stage1/modules/clean-code/clean-code-s1e1.md](https://github.com/rolling-scopes-school/tasks/blob/master/stage1/modules/clean-code/clean-code-s1e1.md)
2. Screenshot: 
![image](https://user-images.githubusercontent.com/1158080/171065473-3461b8fe-c3b8-4bf0-ac17-30f94daf10ab.png)
3. Deploy: [https://yal0.github.io/clean-code-s1e1/index.html](https://yal0.github.io/clean-code-s1e1/index.html)
4. Done 26.10.2022 / deadline 01.11.2022
5. Score: 45 / 45

yal0 committed:
refactor: remove extra spaces and tabs
according to rule 1.1. Отступы in html-and-css.md

yal0 committed:
refactor: uppercase changed to lowercase
according to rule 1.2. Нижний регистр написания in html-and-css.md

yal0 committed:
refactor: single quotes changed to double quotes
according to rule 1.3. Кавычки в HTML/CSS документе in html-and-css.md

yal0 committed:
refactor: add formatting to html
according to rule 2.1. Форматирование in html-and-css.md

yal0 committed:
refactor: add <!DOCTYPE html>
according to rule 2.2. Тип документа / Document Type in html-and-css.md

yal0 committed:
refactor: delete mnemonics
according to rule 2.3. Символы-мнемоники in html-and-css.md

yal0 committed:
refactor: add wrapping long lines
according to rule 2.5. HTML Line-Wrapping in html-and-css.md

yal0 committed:
refactor: fix selectors naming in css
according to rule 3.1. Единый стиль именования селекторов in html-and-css.md

yal0 committed:
refactor: add ; in css
according to rule 3.7. Точка с запятой после свойств in html-and-css.md

yal0 committed:
refactor: add ; in css
according to rule 3.7. Точка с запятой после свойств in html-and-css.md

yal0 committed:
refactor: add semantic tags
according to rule 1.1. Семантика in html-and-css-extended.md

yal0 committed:
refactor: remove tag selectors
according to rule 3.4. Теговые селекторы in html-and-css.md

yal0 committed:
refactor: remove tag selectors
according to rule 3.4. Теговые селекторы in html-and-css.md

yal0 commented:
1. Task: [https://github.com/rolling-scopes-school/tasks/blob/master/stage1/modules/clean-code/clean-code-s1e1.md]()
2. Screenshot: 
![image](https://user-images.githubusercontent.com/1158080/171065473-3461b8fe-c3b8-4bf0-ac17-30f94daf10ab.png)
3. Deploy: []()
4. Done 31.05.2022 / deadline 31.05.2022
5. Score: 45 / 45

yal0 committed:
refactor: use BEM
according to rule 2.1. БЭМ in html-and-css-extended.md

yal0 committed:
refactor: use BEM
according to rule 2.1. БЭМ in html-and-css-extended.md

yal0 committed:
refactor: use BEM
according to rule 2.1. БЭМ in html-and-css-extended.md

yal0 committed:
refactor: delete  type from <script>
according to rule 2.4. Атрибут 'type' in html-and-css.md

Walqer commented:
Привет, у нас согласно правилам коммиты на английском языке принято оформлять :)

yal0 committed:
refactor: uppercase changed to lowercase
according to rule 1.2. Нижний регистр написания in html-and-css.md

yal0 committed:
refactor: remove tag selectors
according to rule 3.4. Теговые селекторы in html-and-css.md

yal0 committed:
refactor: single quotes changed to double quotes
according to rule 1.3. Кавычки в HTML/CSS документе in html-and-css.md

yal0 committed:
refactor: delete  type="text/css"
according to rule 2.4. Атрибут 'type' in html-and-css.md

yal0 committed:
refactor: uppercase changed to lowercase
according to rule 1.2. Нижний регистр написания in html-and-css.md

yal0 committed:
refactor: add spaces
according to rule 3.6. Пробел после названий свойств in html-and-css.md

yal0 committed:
refactor: change class names in css
according to rule 3.2. Значимые названия идентификаторов и классов in html-and-css.md

yal0 committed:
refactor: remove tag selectors
according to rule 3.4. Теговые селекторы in html-and-css.md

yal0 committed:
refactor: remove tag selectors
according to rule 3.4. Теговые селекторы in html-and-css.md

yal0 committed:
refactor: add line breaks to css
according to rule 3.5. Отступы в блоках in html-and-css.md

yal0 committed:
refactor: add semantic tags
according to rule 1.1. Семантика in html-and-css-extended.md

yal0 committed:
refactor: add alt to img
according to rule 1.2. Альтернатива для мультимедиа in html-and-css-extended.md

yal0 committed:
refactor: use BEM
according to rule 2.1. БЭМ in html-and-css-extended.md

yal0 committed:
refactor: add line breaks for group selectors
according to rule 3.8. Разделение селекторов и свойств in html-and-css.md
